metadata/AustralianInvoices__7__metadata.json

```diff
@@ -20,7 +20,5 @@
     "minAIFabricVersion": "v21.10",
     "languageVersion": 2,
     "version": 7,
-    "baseImageCpu": "python37du:v21.8.1-2",
```

Comment:
Don;t we need this?

Reply:
baseImageCpu and baseImageGpu?

Reply:
How do we know which baseImage to use then?

Reply:
based on mlPackageLanguage?

Reply:
already part of changelog, was here for testing. Will avoid few db calls if we remove